fem/transfer.cpp

```diff
@@ -1860,6 +1860,21 @@ L2ProjectionGridTransfer::L2ProjectionH1Space::AllocR()
    return R_local;
 }
 
+L2ProjectionGridTransfer::L2ProjectionH1Space::~L2ProjectionH1Space()
+{
+
+   delete M_LH_local_op;
```

Comment:
Is this not a double free since `transfer.cpp:1231` assigns the ownership of `M_LH_local_op` to the `M_LH`?

Reply:
It looks like `transfer.cpp:1341` also assigns the ownership of `M_LH_local_op` to the `TripleProductOperator` stored in `M_LH`.

Reply:
I think you are right!